src/pages/blog/articles/_article-card.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Flex } from 'components/containers'
+import { Header, Text } from 'components/elements'
+import { localize, LocalizedLink } from 'components/localization'
+import device from 'themes/device'
+
+const ArticleCardWrapper = styled(Flex)`
+    max-width: 384px;
+    flex-direction: column;
+    text-decoration: none;
+    position: relative;
+    height: auto;
+    border-radius: 8px;
+    border: 1px solid var(--color-grey-8);
+    background: var(--color-white);
+    transition: transform 0.3s;
+    overflow: hidden;
+    cursor: pointer;
+
+    &:hover {
+        transform: translateY(-1.1rem) scale(1.02);
+    }
+`
+
+const ImageWrapper = styled.div`
+    height: 200px;
+    width: 384px;
+    position: relative;
+    z-index: 1;
+    overflow: hidden;
+
+    @media ${device.mobileL} {
+        width: 100%;
+    }
+`
+
+const StyledCategories = styled(Text)`
+    width: fit-content;
+    border-radius: 8px;
+    background-color: var(--color-blue-10);
+    color: var(--color-blue-9);
+    padding: 1px 8px;
+    line-height: 20px;
+    margin: 0 8px 8px 0;
+`
+
+const ContentWrapper = styled.div`
+    padding: 16px 24px;
+`
+
+const RedirectLink = styled(LocalizedLink)`
+    text-decoration: none;
+`
+
+const ArticleCard = ({ item }) => {
+    return (
+        <RedirectLink to={`/blog/articles/${item.slug}`}>
+            <ArticleCardWrapper>
+                <ImageWrapper>
+                    <img
+                        src={item.image}
+                        alt={localize('Video card')}
+                        width="100%"
+                        style={{ objectFit: 'contain' }}
+                    />
+                </ImageWrapper>
+                <ContentWrapper>
+                    <Flex jc="flex-start" fw="wrap">
```

Comment:
can we set `margin-bottom` in this component rather than setting the margin inside `StyledCategories `?

Reply:
This is because in the event that the category gets super long and causes the 2nd or 3rd one to wrap on the next line, it'll have a margin in between the 2 categories on 2 different lines.